infrastructure/quick-deploy/localhost/storage.tf

```diff
@@ -39,6 +40,47 @@ module "mongodb" {
   persistent_volume = null
 }
 
+module "mongodb_sharded" {
+  count     = can(coalesce(var.mongodb_sharding)) ? 1 : 0
+  source    = "./generated/infra-modules/storage/onpremise/mongodb-sharded"
+  namespace = local.namespace
+
+  mongodb = {
+    image                 = var.mongodb.image_name
```

Comment:
How do you manage the sharded image?

Reply:
With the default values in the module called

Reply:
Shouldn't you use the same trick as for the GCP and AWS deployment?